context/AuthContext.tsx

```diff
@@ -10,12 +11,9 @@ interface AuthContextType {
 // Create a context with a default value
 const AuthContext = createContext<AuthContextType | undefined>(undefined);
 
-/**
- * Will wrap the content with this AuthProvider so that we can use context to check
- * the authentication state, and to update the state
- * @param param0 
- * @returns 
- */
+
+/*Will wrap the content with this AuthProvider so that we can use context to check
+the authentication state, and to update the state*/
```

Comment:
Still doing checks using NextJS's api endpoint to verify that the user is still authenticated. Not sure if this is still required, since the backend will eventually be regularly checking auth state on every request.